Crashlytics/Crashlytics/FIRCrashlytics.m

```diff
@@ -391,6 +395,8 @@ - (void)recordOnDemandExceptionModel:(FIRExceptionModel *)exceptionModel {
 
 - (void)onSessionChanged:(FIRSessionDetails *_Nonnull)session {
   FIRCLSDebugLog(@"Session ID changed: %@", session.sessionId);
+
+  [self.managerData.contextManager updateAppQualitySessionId:session.sessionId];
```

Comment:
Do you want to send a copy of it rather than the received object?

Reply:
That's a great idea